vcpkg/ports/qtmultimedia/ffmpeg-compile-def.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/plugins/multimedia/ffmpeg/CMakeLists.txt b/src/plugins/multimedia/ffmpeg/CMakeLists.txt
+index 77c459a..af5229e 100644
+--- a/src/plugins/multimedia/ffmpeg/CMakeLists.txt
++++ b/src/plugins/multimedia/ffmpeg/CMakeLists.txt
+@@ -273,7 +273,7 @@ if(BUILD_SHARED_LIBS)
+ else()
+     foreach(ffmpeg_lib IN LISTS ffmpeg_libs)
+         qt_internal_add_target_include_dirs(QFFmpegMediaPlugin ${ffmpeg_lib})
+-        target_include_directories(QFFmpegMediaPlugin PRIVATE
++        target_compile_definitions(QFFmpegMediaPlugin PRIVATE
```

Comment:
see also https://github.com/microsoft/vcpkg/pull/42080/files#diff-8fdc8f7c211644a960aaf73dcd681d427729564f43c5665c7b20408b63699e22

Reply:
@m-kuhn , windows fail to compile, linker complaining about missing ffmpeg functions. Is this linked to that?

Reply:
not sure about it if you say windows.
I was expecting problems on android and hence that they would also need to be fixed in the linked PR. And ideally we have an upstreamable patch.